test/libsolidity/syntaxTests/events/overloading_in_contract.sol

```diff
@@ -0,0 +1,5 @@
+contract A {
+    event X();
+    event X(uint);
```

Comment:
Can you add another test case with `indexed`?

Reply:
you mean `()` and `(uint indexed)`? There is already one with `(uint, uint)` and `(uint, uint indexed)`